app/javascript/dashboard/modules/widget-preview/components/Widget.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="widget-wrapper">
+    <WidgetHead :config="getWidgetHeadConfig" />
+    <WidgetBody />
+    <WidgetFooter />
+  </div>
+</template>
+
+<script>
+import WidgetHead from './WidgetHead';
+import WidgetBody from './WidgetBody';
+import WidgetFooter from './WidgetFooter';
+
+export default {
+  name: 'Widget',
+  components: {
+    WidgetHead,
+    WidgetBody,
+    WidgetFooter,
+  },
+  props: {
+    welcomeHeading: {
+      type: String,
+      default: 'Hi There,',
+    },
+    welcomeTagLine: {
+      type: String,
+      default: '',
+    },
+    websiteName: {
+      type: String,
+      default: '',
+      required: true,
+    },
+    websiteDomain: {
+      type: String,
+      default: '',
+    },
+    logo: {
+      type: String,
+      default: '',
+    },
+    isExpanded: {
+      type: Boolean,
+      default: true,
+    },
+    isOnline: {
+      type: Boolean,
+      default: true,
+    },
+    replyTime: {
+      type: String,
+      default: 'few minutes',
+    },
+  },
+  computed: {
+    // eslint-disable-next-line func-names
+    getWidgetHeadConfig: function() {
+      return {
+        welcomeHeading: this.welcomeHeading,
+        welcomeTagLine: this.welcomeTagLine,
+        websiteName: this.websiteName,
+        websiteDomain: this.websiteDomain,
+        logo: this.logo,
+        isExpanded: this.isExpanded,
+        isOnline: this.isOnline,
+        replyTime: this.replyTime,
+      };
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+$tailwind-slate-50: #f4f6fb;
+.text-lg {
+  font-size: var(--font-size-default);
+}
+.widget-wrapper {
+  width: 400px;
```

Comment:
Can we keep the wrapped width fluid ? Widget preview will be loaded in settings page.

Reply:
I have removed the width